webapp/client/src/components/FormFieldDropDown.js

```diff
@@ -0,0 +1,101 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import FormFieldScaffolding from "./FormFieldScaffolding";
+import FieldLabelForSeparatedFields from "./FieldLabelForSeparatedFields";
+import selectIcon from "../assets/icons/select_icon.svg";
+
+const DropDown = styled.div`
+  & label {
+    display: block;
+    width: 30px;
+    height: 30px;
+    cursor: pointer;
+    background: white;
+    position: absolute;
+  }
+
+  & .steuerlotse-select {
+    border: 2px solid var(--border-color);
+    border-radius: 0;
+    background-image: url(${selectIcon});
+    background-repeat: no-repeat;
+    background-size: 0.75rem;
+    min-height: 55px;
+  }
+
+  & .steuerlotse-select:focus {
+    border: 2px solid var(--focus-border-color);
+    box-shadow: 0 0 0 2px var(--focus-color);
+  }
+`;
+
+function FormFieldDropDown({
+  fieldName,
+  fieldId,
+  values,
+  defaultValue,
+  preselectedValue,
+  required,
+  autofocus,
+  label,
+  details,
+  errors,
+  onChangeHandler,
+}) {
+  return (
+    <FormFieldScaffolding
+      {...{
+        fieldName,
+        errors,
+        cols: "6",
+      }}
+      hideLabel
+      render={() => (
+        <DropDown>
+          <FieldLabelForSeparatedFields {...{ label, fieldId, details }} />
+          <select
+            id={fieldId}
+            className="custom-select steuerlotse-select"
+            input_req_err_msg="Bundesland auswählen"
```

Comment:
Can we implement custom validity with React? Does something roughly like this work?
```suggestion
            onInvalid={(e) => e.target.validity.valueMissing ? "Bundesland auswählen" : ""
```

If it does, we could factor it out into a helper…

Reply:
Thank you so much for pointing that out. I just copied that over from the component in the backend. 

I had another look into this and I think that we could get rid of that completely. We do not use that because we set `novalidate` on all our forms which prevents client validation. We just use the validation messages from the backend. The `input_req_err_msg` has been dead code as far as I can tell. In the future we might want to move that also to the frontend. However, I think it might make more sense to use one of the libraries that we looked into for that.

That said, while looking into it I discovered that it would work if you do something like:
```react
onInvalid={(e) => e.target.setCustomValidity("Bundesland auswählen")
```

TLDR: We can get rid of that because the backend will take care of this.

Reply:
Awesome. Deleted code is the best code :)